lib/ezutils/classes/ezexecution.php

```diff
@@ -177,7 +177,7 @@ static public function registerShutdownHandler( $documentRoot = false )
      * @params Exception the exception
      * @return void
      */
-    static public function defaultExceptionHandler( Exception $e )
+    static public function defaultExceptionHandler( Throwable $e )
```

Comment:
This will break on PHP 5 since it doesn't have `Throwable` class. Typehint should probably be removed altogether.